calico-cloud_versioned_docs/version-19-1/get-started/connect/install-helm.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Steps to connect your cluster to Calico Cloud.
+title: Install using Helm
+---
+
+# Connect a cluster to Calico Cloud using Helm
+
+You can connect your cluster to Calico Cloud by generating a unique Helm command in the Calico Cloud Manager UI and running it on your cluster.
+With Helm installations you can also set optional installation parameters to enable certain features.
+
+## Prerequisites
+
+* You have an active Calico Cloud account. You can sign up for a 14-day free trial at [calicocloud.io](https://calicocloud.io).
+* You are signed in to the Calico Cloud Manager UI as a user with the **Owner**, **Admin**, or **DevOps** role.
+* You have at least one cluster that meets our [system requirements](requirements/system-requirements.mdx).
+* You have kubectl access to the cluster.
+* You have installed Helm 3.0 or later on your workstation.
+
+## Install Calico Cloud using Helm
+
+1. From the **Managed Clusters** page, click **Connect Cluster**.
+1. In the **Connect a Cluster** dialog, enter a **Cluster Name** and select a **Cluster Type**.
+1. (Optional) If you must install a specific older release, select the Calico Cloud version you want to install.
+   We always recommend the latest version, which is installed by default.
+1. Click **Advanced Options**, and then select **Install via helm**
+1. Click **Connect** to generate a unique Helm installation command. Copy the command.
+
+   ```bash title="Example of generated Helm installation command"
+   helm repo add calico-cloud https://installer.calicocloud.io/charts --force-update && helm upgrade --install calico-cloud-crds calico-cloud/calico-cloud-crds --namespace calico-cloud --create-namespace && helm upgrade --install calico-cloud calico-cloud/calico-cloud --namespace calico-cloud --set apiKey=ryl34elz8:5kdv6siag:ifk1uwruwlgp7vzn7ecijt5zjbf5p9p1il1ag8877ylwjo4muu19wzg2g8x5qa7x --set installer.clusterName=my-cluster --set installer.calicoCloudVersion=v19.1.0
+   ```
+1. (Optional) To make changes to what features are enabled during installation, paste the command to a text editor and append the `--set` option any of the following key-value pairs.
```

Comment:
@jaderhs Remind me, are some of these one-way options? If I set these to Disabled during installation, can I change that later in the UI or by other means?

Reply:
> @jaderhs Remind me, are some of these one-way options? If I set these to Disabled during installation, can I change that later in the UI or by other means?

You may change any of these parameters freely between Enabled/Disabled during install or reinstall.

The only way to set is appending them to a helm command for install / reinstall. The helm command can be the automated helm install command or the commands you get when connecting a cluster using helm or reinstalling Calico Cloud (on a cluster that was installed using helm).

There won't be a way to set these parameters using the UI.